app/models/user.rb

```diff
@@ -1251,10 +1251,21 @@ def serializable_hash(options = nil)
     # NOTE: doing deep_dup is necessary here to avoid changing the inner values
     # of the freezed variables (which would leak PII)!
     default_options = DEFAULT_SERIALIZE_OPTIONS.deep_dup
+
+    include_email = options&.[](:include_email)
+    exclude_deprecated = options&.[](:exclude_deprecated)
```

Comment:
Hm, I'm not a big fan of this way of nil-safe calls to the `[]` method.
AI suggests the following alternatives:
- `include_email, exclude_deprecated = options&.values_at(:include_email, :exclude_deprecated)`
- `include_email = options&.dig(:include_email)`
- `include_email = options&.fetch(:include_email, false)`